charts/cloudnative-pg/values.yaml

```
@@ -139,6 +139,16 @@ affinity: {}
 monitoring:
   # -- Specifies whether the monitoring should be enabled. Requires Prometheus Operator CRDs.
   podMonitorEnabled: false
+  grafanaDashboard:
+    enabled: false
+    # -- Allows overriding the namespace where the dashboard is installed.
+    namespace: ""
+    # -- The name of the configmap containing the dashboard.
+    configMapName: "cnpg-grafana-dashboard"
+    # -- Label that config maps with dashboards should have to be added
+    sidecarLabel: "grafana_dashboard"
+    # -- Label value that config maps with dashboards should have to be added
```

I get you were only copying the upstream ones, but I prefer to be consistent with casing on resources names and it didn't sound right the upstream version.
```suggestion
    # -- Label value that ConfigMaps should have to be loaded as dashboards.
```